Database/Corrections/TBC/tbcQuestFixes.lua

```diff
@@ -729,6 +729,9 @@ function QuestieTBCQuestFixes:Load()
         [10231] = {
             [questKeys.triggerEnd] = {"Beat Down \"Dirty\" Larry and Get Information", {[zoneIDs.SHATTRATH_CITY]={{43.86,27.97},},}},
         },
+        [10264] = {
+            [questkeys.preQuestSingle] = {10552},
```

Comment:
I think the better fix is to add reputation requirements to The Scryers and change the prequest to be either 10551 or 10552.

Technically you could side with the Aldor, immediately change your mind and work your way towards the Scryers. Then you could still complete 10264 even though you didn't do 10552 but 10551.